assets/Dockerfiles/nginx.conf

```diff
@@ -0,0 +1,40 @@
+worker_processes auto;
+user             nginx;
+pid              /var/run/nginx.pid;
+error_log        /var/log/nginx/error.log info;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    access_log /var/log/nginx/access.log;
+
+    server {
+        listen              *:80;
+        listen              *:443 ssl;
+        server_name         httpbin.org;
+
+        ssl_protocols       TLSv1.2 TLSv1.3;
+        ssl_certificate     /etc/nginx/hostname_cert.pem;
+        ssl_certificate_key /etc/nginx/hostname_key.pem;
+
+        location / {
+            proxy_pass http://httpbin;
+        }
+    }
+
+    server {
+        listen              *:80;
+        listen              *:443 ssl;
+        server_name         some_authority;
```

Comment:
This makes `t/03-proxy_wasm/sdks/002-go_sdk/005-dispatch_call_on_tick.t` pass locally (`act --pull=false`) but not on CI for now